Source/Request.swift

```diff
@@ -308,7 +308,12 @@ extension Request: CustomDebugStringConvertible {
                 let cookies = cookieStorage.cookies(for: url), !cookies.isEmpty
             {
                 let string = cookies.reduce("") { $0 + "\($1.name)=\($1.value);" }
-                components.append("-b \"\(string.substring(to: string.characters.index(before: string.endIndex)))\"")
+
+                #if swift(>=3.2)
+                    components.append("-b \"\(string[..<string.index(before: string.endIndex)])\"")
```

Comment:
I think we will still want the `before:` call. `substring(to:)` gets you every character except for the one specified so in this case it seems to be stripping out the last character in the string.